forwards_paper.tex

```diff
@@ -212,32 +217,35 @@ \subsection*{Simulation benchmarks}
 coalescent \citep{Neuhauser1997-nn}.  Therefore, these selected mutations must be explicitly tracked in our forward simulation
 and the time savings due to pedigree recording come from not having to record \textit{neutral} mutations.
 
+To demonstrate the method's flexibility and to test it in different contexts,
+we implemented these simulations separately using two different simulation engines:
+\begin{enumerate}
 
-We ran all benchmarks on an Ubuntu Linux (version 16.04) system with two 2.6 Gigahertz Intel E5-2650 CPU with
-hyperthreading enabled.  We ran one simulation at a time and the machine was under minimal load otherwise.  We used GNU
-parallel \citep{Tange2011a} to kill
-any simulation that did not finish within 72 hours.  We used the Linux \texttt{time} command to record run time and peak
-memory usage of each replicate and our simulation scripts separately recorded the time spent in various steps of each
-run.
-We varied $N$ and $\rho$ and ran a single replicate of all $(N,\rho)$
-combinations (the actual values differed for \simupop{} and \fwdpp{}, see below).
+    \item 
+        In C++, using \fwdpp{} library functions and interface code (details below),
+        using a continuum-sites model for both mutation and recombination.
+        % $\mu$ and $r$ were the means of Poisson processes,
+        % and mutations used the infinite-sites model \citep{Kimura1969-uc}.
+        % We varied $N \in (1000, 10000, 50000)$, $\rho \in (10^3, 10^4, 10^5)$.
 
-\subsubsection*{The \fwdpp{} implementation}
+    \item 
+        In Python, using the \simupop{} library interfacing with a custom Python library (\ftprime{}),
+        using a discrete but large number of sites.
+        % $\mu$ and $r$ define per-base-pair mutation rate and we
+        % defined the length of the simulated sequence, in base pairs, as $L = \tfrac{\rho}{4N \mu}$.
+        % We varied $N \in (1000, 10000)$, $\rho \in (10^3, 10^4)$.
 
-We implemented Wright-Fisher simulations in C++ using \fwdpp{} library
-functions and interface code that records data into the \msprime{} Tables API (see below).
-In these simulations, $\mu$ and $r$ are the means of a Poisson process and we used the infinitely-many sites mutation model
-\citep{Kimura1969-uc}.
-We varied $N \in (1000, 10000, 50000)$, $\rho \in (10^3, 10^4, 10^5)$.
+\end{enumerate}
 
-\subsubsection*{The \simupop{} implementation}
```

Comment:
didn't seem like enough content to warrant a subsection.  although more detail could go in a subsection of Methods.